master-nonlatent/master.cfg

```diff
@@ -114,7 +114,13 @@ c["workers"].append(aix_worker)
 
 # Docker Library
 dockerlibrary_worker = mkWorker(
-    os.getenv("MASTER_NONLATENT_DOCKERLIBRARY_WORKER", default="bb-rhel8-docker"), properties={"jobs": 1, "save_packages": True}
+    os.getenv("MASTER_NONLATENT_DOCKERLIBRARY_WORKER", default="bb-rhel8-docker"),
+    properties={
+        "jobs": 1,
+        "save_packages": os.getenv("ENVIRON") == "PROD",
+        "environ": os.getenv("ENVIRON"),
```

Comment:
Since these are env vars there isn't a need to set them up as properties and we can use arguments/variables instead. I think that this option would be cleaner

Reply:
ok. Changing docker-library-manifest.